modeling-cmds/src/each_cmd.rs

```diff
@@ -189,6 +189,19 @@ pub struct EntityGetDistance {
     pub distance_type: DistanceType,
 }
 
+/// Create a linear patter using this entity
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct EntityLinearPattern {
+    /// ID of the entity being copied.
+    pub entity_id: Uuid,
+    /// Axis along which to make the copites
+    pub axis: Point3d<f64>,
```

Comment:
I'm confused, how does a 3D point represent an axis?

Reply:
The axis does not have to correspond to a global axis. For example, if a user wanted to duplicate an object along a 45 degree angle (that moves in both the x and z direction), they could specify the axis to be (1, 0, 1)

Reply:
And the axis is normalized before being used!

Reply:
Does this support using an existing face as an axis? E.g. if I want to duplicate along 45 degrees XZ globally, or 45 degrees XZ in reference to some face of my model, how do we support that?

Reply:
Answering myself, no it doesn't, because this is only for solid3ds which cannot be along a face.